tests/python/openassetio_mediacreation/test_imports.py

```diff
@@ -52,3 +52,6 @@ def test_importing_ManagedTrait_succeeds(self):
 
     def test_importing_ResolvesFutureEntitiesTrait_succeeds(self):
         from openassetio_mediacreation.traits.managementPolicy import ResolvesFutureEntitiesTrait
+
+    def test_importing_DisplayNameTrait_succeeds(self):
+        from openassetio_mediacreation.traits.identity import DisplayNameTrait
```

Comment:
We generally have a policy of "don't test what trait gen does", feels like we should define this a bit more formally going forward. Do we think we need to double these things up more generally? I kind of saw the import tests as "did we put what we think we put in the YAML"? But its only partially doing that.

Reply:
It would be great if we could simply remove even this test.  I think though, whilst TraitGen is still a little new and not battle tested, it would be worth checking that at least there are no syntax errors preventing `import`/`#include`.

Reply:
Added [here](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/30/commits/782e163b5a93807d270945dfff79707f2444b108).